.github/workflows/test.yml

```
@@ -0,0 +1,45 @@
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          target/
+        key: ${{ runner.os }}-cargo-quick-${{ hashFiles('**/Cargo.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-cargo-
+        
+    - name: Install PocketIC server
+      uses: dfinity/pocketic@main
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)